dds_cli/__main__.py

```diff
@@ -993,37 +757,68 @@ def archive(click_ctx, username, project):
         sys.exit(1)
 
 
-@status.command()
-@common_options
-@common_options_project
+# ACCESS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ACCESS #
+@project_group_command.group(name="access")
+@click.pass_obj
+def project_access(_):
+    """Manage project statuses."""
```

Comment:
Maybe a typo? Should be managing access instead of status here.

Reply:
Fixed!